test/indent-parser.test.ts

```diff
@@ -0,0 +1,38 @@
+
+import {
+  parse,
+} from '../lib/indent-parser';
+
+const fullPacketLockContents = `
+  remote: https://nuget.org/api/v2
+    FSharp.Core (4.0.0.1)
+    Newtonsoft.Json (7.0.1)
+      UnionArgParser (0.6.3)`;
+
+describe('indent parse', () => {
+  it('correctly returns null children under dependencies', () => {
+
+    const output = parse ('');
+    expect(output).toEqual({ dependencies: null });
+  });
+
+  it('correctly returns parsed indented tree with whitespace', () => {
+
+    const output = parse (fullPacketLockContents);
+    console.log(JSON.parse(JSON.stringify(output)));
+    expect(output).toEqual({ dependencies:
```

Comment:
Please can we keep the fixtures separate? It is hard to read this and super hard to add larger fixtures inline like this